diff --git a/node_modules/sequelize/lib/sequelize.js b/node_modules/sequelize/lib/sequelize.js
index 05604cd..eb7c006 100644
--- a/node_modules/sequelize/lib/sequelize.js
+++ b/node_modules/sequelize/lib/sequelize.js
@@ -182,7 +182,7 @@ class Sequelize {
       config = {};
       options = username || {};
 
-      const urlParts = url.parse(arguments[0], true);
+      const urlParts = url.parse(options.use_env_variable, true);
 
       options.dialect = urlParts.protocol.replace(/:$/, '');
       options.host = urlParts.hostname;
